tools/dotcms-cli/cli/src/test/java/com/dotcms/cli/command/language/LanguageCommandTest.java

```diff
@@ -93,4 +97,63 @@ void Test_Command_Language_Find() {
             Assertions.assertTrue(output.contains("English"));
         }
     }
+
+    /**
+     * Test the language push command. A new language with tag "es-VE" will be created.
+     */
+    @Test
+    void Test_Command_Language_Push_byTag() {
+        final CommandLine commandLine = getFactory().create();
+        final StringWriter writer = new StringWriter();
+        try (PrintWriter out = new PrintWriter(writer)) {
+            commandLine.setOut(out);
+            final int status = commandLine.execute(LanguageCommand.NAME, LanguagePush.NAME, "--byTag", "es-VE");
+            Assertions.assertEquals(CommandLine.ExitCode.OK, status);
+            final String output = writer.toString();
+            Assertions.assertTrue(output.contains("Spanish"));
+        }
+    }
+
+    /**
+     * Test the language push command using a JSON file as an input. A new language with tag "it-IT" will be created.
+     */
+    @Test
+    void Test_Command_Language_Push_byFile_JSON() throws IOException {
+        final CommandLine commandLine = getFactory().create();
+        final StringWriter writer = new StringWriter();
+        try (PrintWriter out = new PrintWriter(writer)) {
+            //Create a JSON file with the language to push
+            final Language language = Language.builder().languageCode("it-IT").countryCode("IT").language("Italian").country("Italy").build();
+            final ObjectMapper mapper = new ClientObjectMapper().getContext(null);
+            final File targetFile = File.createTempFile("language", ".json");
+            mapper.writeValue(targetFile, language);
+            commandLine.setOut(out);
+            final int status = commandLine.execute(LanguageCommand.NAME, LanguagePush.NAME, "-f", targetFile.getAbsolutePath());
+            Assertions.assertEquals(CommandLine.ExitCode.OK, status);
+            final String output = writer.toString();
+            Assertions.assertTrue(output.contains("Italian"));
+        }
+    }
+
+    /**
+     * Test the language push command using a YAML file as an input. A new language with tag "it-IT" will be created.
+     */
+    @Test
+    void Test_Command_Language_Push_byFile_YAML() throws IOException {
+        final CommandLine commandLine = getFactory().create();
+        final StringWriter writer = new StringWriter();
+        try (PrintWriter out = new PrintWriter(writer)) {
+            //Create a YAML file with the language to push
+            final Language language = Language.builder().languageCode("it-IT").countryCode("IT").language("Italian").country("Italy").build();
+            final ObjectMapper mapper = new YAMLMapperSupplier().get();
+            final File targetFile = File.createTempFile("language", ".yml");
+            mapper.writeValue(targetFile, language);
+            commandLine.setOut(out);
+            final int status = commandLine.execute(LanguageCommand.NAME, LanguagePush.NAME, "-f", targetFile.getAbsolutePath(), "-fmt",
```

Comment:
do We also support `-fmt` ?
I see I used on the other examples `--format `
there's nothing wrong about using `-fmt` actually I think it's quite friendly just wondering if we're currently supporting both` --fromat` and` -fmt `

Reply:
yes, we support both. I'm reusing the OutputOptionMixin. I used `-fmt` on this test, but I could have used `--format` as well